containers/deploy/scripts/docker-compose-data-services.yml

```
@@ -0,0 +1,33 @@
+---
+services:
+  mira-dkg:
+    container_name: mira-dkg
+    image: ghcr.io/darpa-askem/mira:latest
+    networks:
+      - terarium
+    environment:
+      MIRA_NEO4J_URL: 'bolt://neo4j:7687'
+    extra_hosts:
+      - "${local_host_name}:host-gateway"
+
+  beaker:
+    container_name: beaker
+    image: ghcr.io/darpa-askem/beaker-kernel:latest
+    networks:
+      - terarium
+    command: ["python", "-m", "beaker_kernel.service.notebook"]
```

We can use the image on GitHub for simplicity, and the command is part of the image already.
```suggestion
  beaker:
    container_name: beaker
    image: ghcr.io/darpa-askem/askem-beaker:latest
    networks:
      - terarium
```